app/controllers/application_controller.rb

```diff
@@ -52,6 +52,7 @@ class ApplicationController < ActionController::Base
   include_concern 'TreeSupport'
   include_concern 'SysprepAnswerFile'
   include_concern 'ReportDownloads'
+  include_concern 'Show'
```

Comment:
Yes, that is true, I put this code here so all other places can reuse these methods, Should we move this to another place? for example to another mixin?